### PR TITLE
New Translation (GUJARATI)

### PR DESCRIPTION
**Pull request by Devrajsinh03**
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>
I have added a new translation in GUJARATI language to make cheatsheet more accessible
<Link to related issue. Type `closes #RELATEDISSUENUMBER` to establish a link.>

### Questions
<If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>

### Next Steps
<If incomplete, create a task list of items that are still being worked on within the Pull Request.>

### Review
<If complete, or ready for :eyes:, use @mentions for quick questions, specific feedback, and progress updates.>
@brianamarie I have added GUJARATI Translation for the cheat-sheet and I myself is a native speaker of it and it is accurate to the best of my knowledge.

### CODE REVIEW

**Comment by Devrajsinh03** [comment]
@parkerbxyz can you please review this code, as I am a native speaker and don't know any other native speakers here. And it is accurate to the best of my knowledge.

File: downloads/gu/github-git-cheat-sheet.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+
+```$ git stash pop```
+
+તાજેતરમાં સાચવેલી ફાઇલ પુનઃપ્રાપ્ત કરો
+
+
+```$ git stash list```
+
+બધા અસ્થાયી રૂપે સાચવેલા ફેરફાર સેટની સૂચિ બનાવો
```
Comment (parkerbxyz) [inline downloads/gu/github-git-cheat-sheet.md:175]:
```suggestion
બધા અસ્થાયી રૂપે સાચવેલ ફેરફાર સેટ ની સૂચિ બનાવો
```

File: downloads/gu/github-git-cheat-sheet.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+
+```$ git log```
+
+વર્તમાન શાખા માટે સંસ્કરણ ઇતિહાસની સૂચિ જુઓ
+
+
+```$ git log --follow [file]```
+
+નામ સહિત, ફાઇલ માટે સંસ્કરણ ઇતિહાસની સૂચિ આપે છે
```
Comment (parkerbxyz) [inline downloads/gu/github-git-cheat-sheet.md:198]:
```suggestion
નામ સાથે, ફાઇલ માટે સંસ્કરણ ઇતિહાસની સૂચિ આપે છે
```

File: downloads/gu/github-git-cheat-sheet.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+
+```$ git fetch [remote]```
+
+રિપોઝીટરી બુકમાર્ક્સમાંથી તમામ ઇતિહાસ ડાઉનલોડ કરો
+
+
+```$ git merge [remote]/[branch]```
+
+બુકમાર્ક શાખાને વર્તમાન સ્થાનિક શાખામાં મર્જ કરો
```
Comment (parkerbxyz) [inline downloads/gu/github-git-cheat-sheet.md:234]:
```suggestion
બુકમાર્ક શાખા વર્તમાન સ્થાનિક શાખામાં મર્જ કરો
```

**Review by parkerbxyz — COMMENTED** [review]
Thanks for taking the time to translate this document and open this pull request, @devarshgandhi! ✨ I left several suggested changes, most of which where suggested by Google Translate. Since you are a native speaker and I am not, feel free to reject any of the suggestions, as they may not be good ones.

**Comment by Devrajsinh03** [comment]
@parkerbxyz I've made necessary they were just upper and lower case changes, the others left are not necessarily and now it looks good. 👍

**Review by parkerbxyz — APPROVED** [review]
